modules/ROOT/pages/servers-cert-renewal.adoc

```diff
@@ -4,35 +4,109 @@ include::_attributes.adoc[]
 endif::[]
 
 image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
-Registering a Mule server in Runtime Manager requires a valid certificate provisioned and signed by Runtime Manager. The certificate ensures secure communications between Runtime Manager and the Runtime Manager agent. 
+Registering a Mule server in Anypoint Runtime Manager requires a valid certificate provisioned
+and signed by Runtime Manager.
+The certificate ensures secure communication between Runtime Manager and the Runtime Manager agent. 
 
-However, certificates expire, and you cannot renew them after they expire. Before a certificate expires, renew it directly from Runtime Manager. You can see the expiration date on the server dashboard.
+When a server certificate expires, Runtime Manager displays `Disconnected` in the *Status* column 
+for the server on the *Servers* page.
+The `mule-agent.log` file displays a message like the following:
 
-A server must be running for you to update its certificate. You can update a certificate for a specific server or multiple servers within a cluster or group.
+----
+com.mulesoft.agent.transport.WSConnection: Failed attempt to connect nro. 1 to the web socket client at mule-manager.anypoint.mulesoft.com: Connection refused.
+----
 
+For applications deployed to the server, Runtime Manager displays `Unknown` in the *Status* column on the *Applications* page.
+
+== View a Certificate Expiration Date
+
+To see the expiration date for a server:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Click the *Servers* tab.
+. Click the server name.
++
+For servers in a server group or cluster, click the server group or cluster name and then click the server name.
++
+The certificate expiration date appears on the server dashboard:
++
+.The arrow shows the *Certificate expiration date* on the *Dashboard* page.
+image::rtm-server-certificate.png[Certificate expiration date on the Dashboard page]
+
+
+== Renew a Certificate Before it Expires 
```

Comment:
```suggestion
== Renew a Certificate before It Expires 
```
It's not the length of the word (except for prepositions) but the type of word, and nouns and pronouns are always init. capped. Note also that when using https://capitalizemytitle.com/, we click the Chicago tab.